zeusd/src/devices/cpu/mod.rs

```diff
@@ -26,14 +26,13 @@ pub struct PackageInfo {
     pub name: String,
     pub energy_uj_path: PathBuf,
     pub max_energy_uj: u64,
-
-    num_wraparounds: RwLock<u64>,
+    pub num_wraparounds: RwLock<u64>,
 }
 
-#[derive(Serialize)]
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
```suggestion
#[derive(Serialize, Debug)]
#[cfg_attr(test, derive(Deserialize))]
```
I think this would work. I'm assuming you need `Deserialize` only for testing. (Move `Debug` into `cfg_attr` as well if you also only needed `Debug` solely for testing).

Reply:
I'm getting

```
error[E0277]: the trait bound `RaplResponse: Deserialize<'_>` is not satisfied
    --> tests/cpu.rs:77:43
     |
77   | ...sponse = serde_json::from_str(&resp.text().await.unwrap())
     |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Deserialize<'_>` is not implemented for `RaplResponse`
```

According to chat gpt:
When you run cargo test, Rust compiles your library code without the test flag and then compiles your tests with the test flag. This means any #[cfg(test)] or #[cfg_attr(test, ...)] in your library code won't have any effect during tests because the library code isn't recompiled with the test flag.